scripts/seed.sql

```diff
@@ -16,6 +16,26 @@ VALUES
   ('Irene', '$2b$10$Uo.576WqbP7l7zcHr2SN7./Dw/iEq9HyZ/7wGVh/pifGSCviyu6Sq'),
   ('Quinn', '$2b$10$d/fCrQl3RBSV7BaSJDq0yel0X2lCoNgo5fpmhxdvPbscek8BKUcNO');
 
+INSERT INTO roles (role_id, name)
+VALUES
+  (0, 'suspended'),
+  (1, 'user'),
+  (2, 'admin');
+
+INSERT INTO users_roles (user_id, role_id)
+VALUES
+  (1, 1),
+  (1, 1),
```

Comment:
Du coup, avec le commit aa77eb575e292630aad19713708dee3ed28396de, cette insertion ne fonctionne plus. 
```suggestion
  (1, 1),
```